### ISSUE TITLE
Sickrage stops responding when rechecking files

### ISSUE THREAD

**Issue by selthius** [SickRage/SickRage#1445]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master / e718ba29b4fd92099d2c2c665ae9c7845a633c23
OS: Linux-4.2.0-27-generic-x86_64-with-Ubuntu-15.10-wily
What you did: Re-check files for Bleach when adding a new season
What happened: Sickrage stopped responding after some time
What you expected: Sickrage rescanning and identifying the new season
Logs:
```
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: Parsed Bleach - S03E20 - Aizen Stands Out! His Terrifying Ambition into Bleach - S03E20 [ABS: [61]] [ANIME VER: -1] [ABD: False] [ANIME: True] [whichReg: ['anime_SxxExx']]
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: 74796: Not creating SQL queue - record is not dirty
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: 74796: Creating episode from /storage/Anime/Series/Bleach/Bleach Season 07/Bleach - 148 [x264.AAC][733B4513].mkv
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: 74796: Creating episode object from /storage/Anime/Series/Bleach/Bleach Season 07/Bleach - 148 [x264.AAC][733B4513].mkv
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: Found episode by absolute number 148 which is S07E17
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: Converted parsed result Bleach - 148 [x264.AAC][733B4513] into Bleach - S07E17 [ABS: [148]] [ANIME VER: 1] [ABD: False] [ANIME: True] [whichReg: ['anime_standard_codec']]
2016-04-11 21:28:58 DEBUG    SHOWQUEUE-REFRESH :: Trying to find bleach season on theTVDB
2016-04-11 21:28:59 DEBUG    SHOWQUEUE-REFRESH :: Found episode by absolute number 7 which is S01E07
2016-04-11 21:28:59 DEBUG    SHOWQUEUE-REFRESH :: Converted parsed result Bleach Season 07 into Bleach Season - S01E07 [ABS: [7]] [ANIME VER: 1] [ABD: False] [ANIME: True] [whichReg: ['anime_bare']]
2016-04-11 21:28:59 DEBUG    SHOWQUEUE-REFRESH :: Parsed /storage/Anime/Series/Bleach/Bleach Season 07/Bleach - 148 [x264.AAC][733B4513].mkv into Bleach Season - S07E17 [ABS: [148]] [ANIME VER: 1] [ABD: False] [ANIME: True] [whichReg: ['anime_standard_codec', 'anime_bare']]
2016-04-11 21:28:59 DEBUG    SHOWQUEUE-REFRESH :: 74796: /storage/Anime/Series/Bleach/Bleach Season 07/Bleach - 148 [x264.AAC][733B4513].mkv parsed to Bleach S07E17
2016-04-11 21:28:59 DEBUG    SHOWQUEUE-REFRESH :: Setter sets location to /storage/Anime/Series/Bleach/Bleach Season 07/Bleach - 148 [x264.AAC][733B4513].mkv
```

I also checked with another show I only have 1 season of, 24, which works flawlessly

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by selthius** [comment]
I think I just narrowed it down some more. I added multiple seasons of another Anime, which it detected perfectly, except for the second part of a double episode was missing. I used sickrage to rename all the episodes and manually edited in the missing episode, but when I did another recheck it stopped responding again. This is the log:
https://gist.github.com/selthius/78b08672f406ba9351f758c9de1aa716

**Comment by miigotu** [comment]
Try to narrow it down to one file that it hangs on.

**Comment by selthius** [comment]
I tried Bleach multiple times, and it always get stuck on the same file, episode 148, but it seems to be regardless of file, as it also happened with Gintama.

**Comment by miigotu** [comment]
How much ram do you have

**Comment by selthius** [comment]
48 GB

**Comment by selthius** [comment]
It happened again while I was asleep. This is the last page or so of the log, please tell me if you need more.

https://gist.github.com/selthius/cd14c84e95d4d4ebbd84c833d2ccd918

**Comment by miigotu** [comment]
Can you try now with the develop branch please?

**Comment by salfter** [comment]
I'm running the most recent release, though I first noticed this problem yesterday.  It went away yesterday when I rolled back maybe 60 or so of the most recent commits.  

I usually reencode files after downloading (especially to convert H.265 to something viewable on my Raspberry Pis), which ends up necessitating a file rescan afterward.  This is what I got from my last attempt, running in the foreground:

```
Not running in daemon mode. PID file creation disabled.
12:01:10 DEBUG::MAIN :: Parsed Show.Name.S02E03.HDTV.XviD-RLSGROUP into Show Name - S02E03 [GROUP: RLSGROUP] [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:10 DEBUG::MAIN :: Found codec for 'Show Name: Ep Name'.
12:01:10 DEBUG::MAIN :: Episode has no release group, replacing it with 'RLSGROUP'
12:01:10 DEBUG::MAIN :: formatting pattern: %SN - S%0SE%0E - %EN -> Show Name - S02E03 - Ep Name
12:01:10 DEBUG::MAIN :: Trying to parse Show Name - S02E03 - Ep Name.ext
12:01:10 DEBUG::MAIN :: Parsed Show Name - S02E03 - Ep Name.ext into Show Name - S02E03 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:10 DEBUG::MAIN :: The name Show Name - S02E03 - Ep Name.ext parsed into Show Name - S02E03 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:10 DEBUG::MAIN :: Checking database structure...sickbeard.db
12:01:10 DEBUG::MAIN :: Checking database structure...cache.db
12:01:10 DEBUG::MAIN :: Checking database structure...failed.db
12:01:10 DEBUG::MAIN :: Checking for shows with tvrage id's, since tvrage is gone
12:01:10 DEBUG::MAIN :: Checking for archived episodes not qualified
12:01:10 DEBUG::MAIN :: Retrieving the last check time from the DB
12:01:10 DEBUG::MAIN :: Loading initial show list
12:01:10 INFO::MAIN :: 80189: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: 74447: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: 72514: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: 81059: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: 81001: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: 79009: Unable to find IMDb show info in the database
12:01:10 INFO::MAIN :: Starting SickRage [master] using '/etc/sickrage/sickrage.ini'
12:01:10 INFO::MAIN :: Forcing web server to port 8081
12:01:10 INFO::TORNADO :: Starting SickRage on http://0.0.0.0:8081/
12:01:10 DEBUG::CHECKVERSION :: Starting new thread: CHECKVERSION
12:01:10 INFO::CHECKVERSION :: Checking for updates using SOURCE
12:01:11 DEBUG::CHECKVERSION :: cur_commit = e718ba29b4fd92099d2c2c665ae9c7845a633c23, newest_commit = e718ba29b4fd92099d2c2c665ae9c7845a633c23, num_commits_behind = 0
12:01:11 DEBUG::CHECKVERSION :: check_for_new_news: Checking GitHub for latest news.
12:01:19 DEBUG::Thread-13 :: Queueing show refresh for Better Call Saul
12:01:20 INFO::SHOWQUEUE-REFRESH :: Performing refresh on Better Call Saul
12:01:20 DEBUG::SHOWQUEUE-REFRESH :: 273181: Loading all episodes from the show directory /mnt/files/video/tv-shows/Better Call Saul
12:01:20 DEBUG::SHOWQUEUE-REFRESH :: 273181: Found files: [u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E10 - Marco.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E05 - Alpine Shepherd Boy.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E03 - Amarillo.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E04 - Gloves Off.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E01 - Uno.m2ts', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E09 - Pimento.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E04 - Hero.m2ts', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E07 - Inflatable.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E02 - Mijo.m2ts', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E03 - Nacho.m2ts', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E06 - Five-O.m4v', u"/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E06 - Bali Ha'i.m4v", u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E02 - Cobbler.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E05 - Rebecca.m4v', u'/mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E08 - Rico.m4v']
12:01:20 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v
12:01:20 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode object from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v into Better Call Saul - S02E08 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v parsed to Better Call Saul S02E08
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Transaction with 1 queries executed
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E08 - Fifi-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed Better Call Saul - S02E08 - Fifi into Better Call Saul - S02E08 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode object from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v into Better Call Saul - S01E07 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v parsed to Better Call Saul S01E07
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Transaction with 1 queries executed
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S01E07 - Bingo-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed Better Call Saul - S01E07 - Bingo into Better Call Saul - S01E07 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode object from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v into Better Call Saul - S02E01 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v parsed to Better Call Saul S02E01
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Transaction with 1 queries executed
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E01 - Switch-thumb.jpg exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed Better Call Saul - S02E01 - Switch into Better Call Saul - S02E01 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: Creating episode object from /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Parsed /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v into Better Call Saul - S02E09 [ABD: False] [ANIME: False] [whichReg: ['standard']]
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v parsed to Better Call Saul S02E09
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.mkv
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: 273181: The old episode had a different file associated with it, re-checking the quality using the new filename /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Setter sets location to /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.nfo exists: True
12:01:21 DEBUG::SHOWQUEUE-REFRESH :: Checking if /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed-thumb.jpg exists: False
Segmentation fault
```

Basically, Better Call Saul - S02E09 - Nailed.mkv was reencoded (H.264 video, HE-AAC audio, resolution and framerate unchanged) and replaced with Better Call Saul - S02E09 - Nailed.m4v.  Something's causing it to fall over with a segfault.

**Comment by miigotu** [comment]
can you try on the command line:
`mediainfo /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v`

**Comment by salfter** [comment]
```
General
Complete name                            : /mnt/files/video/tv-shows/Better Call Saul/Better Call Saul - S02E09 - Nailed.m4v
Format                                   : MPEG-4
Format profile                           : Base Media / Version 2
Codec ID                                 : mp42
File size                                : 394 MiB
Duration                                 : 49mn 4s
Overall bit rate                         : 1 122 Kbps
Movie name                               : PSArips.com | Better.Call.Saul.S02E09.720p.HDTV.2CH.x265.HEVC-PSA
Encoded date                             : UTC 2016-04-12 17:35:32
Tagged date                              : UTC 2016-04-12 17:35:32
Writing application                      : HandBrake 0.10.5 2016040400

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4.1
Format settings, CABAC                   : Yes
Format settings, ReFrames                : 4 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 49mn 4s
Bit rate                                 : 989 Kbps
Width                                    : 1 280 pixels
Height                                   : 720 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 23.976 fps
Minimum frame rate                       : 23.974 fps
Maximum frame rate                       : 23.981 fps
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.045
Stream size                              : 347 MiB (88%)
Writing library                          : x264 core 142
Encoding settings                        : cabac=1 / ref=3 / deblock=1:0:0 / analyse=0x3:0x113 / me=hex / subme=7 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=1 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=6 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=2 / keyint=240 / keyint_min=24 / scenecut=40 / intra_refresh=0 / rc_lookahead=40 / rc=crf / mbtree=1 / crf=21.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=62500 / vbv_bufsize=78125 / crf_max=0.0 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:1.00
Encoded date                             : UTC 2016-04-12 17:35:32
Tagged date                              : UTC 2016-04-12 17:35:32
Color range                              : Limited
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709

Audio
ID                                       : 2
Format                                   : AAC
Format/Info                              : Advanced Audio Codec
Format profile                           : HE-AAC / LC
Codec ID                                 : 40
Duration                                 : 49mn 4s
Bit rate mode                            : Constant
Bit rate                                 : 128 Kbps
Channel(s)                               : 2 channels
Channel positions                        : Front: L R
Sampling rate                            : 48.0 KHz / 24.0 KHz
Compression mode                         : Lossy
Stream size                              : 44.9 MiB (11%)
Title                                    : Stereo / Stereo
Encoded date                             : UTC 2016-04-12 17:35:32
Tagged date                              : UTC 2016-04-12 17:35:32
```

**Comment by miigotu** [comment]
If you are comfortable changing some code to narrow this down it will help (I cannot reproduce this problem)

https://github.com/SickRage/SickRage/blob/master/sickrage/helper/media_info.py#L119

Stop SR, and edit `SickRage/sickrage/helper/media_info.py`

Change line 119 from:
```python
for method in [_mediainfo_screen_size, _mkv_screen_size, _avi_screen_size]:
```
To:
```python
for method in [_mkv_screen_size, _avi_screen_size]:
```

See if that fixes the problem. The pymediainfo library is the only new addition so it feels like it must be the problem.

**Comment by salfter** [comment]
Tried that...it loses the resolution information from the old file and sets it to "unknown," but at least it's not crashing.

**Comment by miigotu** [comment]
Well ofc it would, we dont have a parser for m4v without mediainfo. I will see about changing that implementation to another mediainfo wrapper
